src/main/java/at/hannibal2/skyhanni/utils/repopatterns/RepoPatternManager.kt

```diff
@@ -97,8 +98,9 @@ object RepoPatternManager {
      * Reload patterns in [usedKeys] from [regexes] or their fallbacks.
      */
     private fun reloadPatterns() {
+        val localLoading = config.forceLocal.get() || LorenzUtils.isInDevEnviromen()
```

Comment:
Add this to the RepoPatternMaanager as a val get. A more central place to get the value from. And also I have an open PR that would needs this value.